src/routes/auth.ts

```diff
@@ -131,7 +131,16 @@ router.post("/login", async (req: Request, res: Response) => {
       message: "Email and password required",
     });
 
-  let user = await querySelectUserByEmail(email);
+  let user;
+  try {
+    user = await querySelectUserByEmail(email);
+  } catch (error) {
+    return res.status(503).json({
+      ok: false,
+      message: "Unable to connect to the database",
```

Comment:
should we used a more standard message that doesn't give info to the end user like, 'we are having problems try later or contact the administrator' type of thing?

Reply:
What do you think about "Something went wrong at our end, try again later."?